### DISCUSSION TITLE
跳转链接待修正

### DISCUSSION THREAD

**Discussion by channely** [galacean/discussion#2] [Editor]
打开[编辑器](https://galacean.antgroup.com/editor)后

<img width="248" alt="截屏2023-04-26 14 12 36" src="https://user-images.githubusercontent.com/3193212/234485403-020b1c49-534c-47c3-97e4-76b47186f7de.png">

- 其中“文档”的跳转，需将 https://oasisengine.cn/#/docs/latest/cn/editor 改为 https://oasisengine.cn/#/docs/latest/cn/editor
- 其中“讨论”的跳转，需将 https://github.com/ant-galaxy/galacean/engine/discussions/categories/editor 改为 https://github.com/galacean/engine/discussions/new?category=editor

**Comment by eyworldwide** [comment]
github 已经支持组织级别的 discussions，所以不太适合放到 engine 仓库的 discussion 下，讨论地址改成：https://github.com/orgs/galacean/discussions/new?category=editor

issue: https://github.com/galacean/editor/issues/1523
